.github/workflows/reusable-e2e.yml

```diff
@@ -53,7 +58,6 @@ jobs:
       - name: Compile Contracts
         run: |
           cd contracts
-          npm run compileSol
```

Comment:
Why remove?

Reply:
this is now run together with the contracts build script (`prebuild`) - pretty neat stuff @0xmad

Reply:
Indeed very neat

Reply:
Right, compilation step is added in pre-hook.